apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/BindingTests.cs

```diff
@@ -0,0 +1,195 @@
+﻿#region
```

Comment:
I think resharper may have removed the license when I did a code cleanup.  fixed